PublicMapping/district-builder-2
# Initial application setup

## Overview

Initial project setup. Based off of https://github.com/azavea/react-redux-typescript-template/

## Demo

![db2_setup](https://user-images.githubusercontent.com/2926237/69743670-cfb13b80-110c-11ea-8def-ccbc8b197d30.png)

## Notes

I'm uncomfortable with adding a bunch of dependencies without dogfooding them -- eg. Redux, `redux-loop`, `axios`, and `ts.data.json` are all added without being used in any meaningful way -- but that's what this issue says to do.

I kept a mostly placeholder `reducers.ts` file for now. There is no `actions.ts` since the app doesn't do anything yet, but we can refer to the template repo once we start to make meaningful things happen in the app for how to use Redux/redux-loop.

I grabbed `logo.png` from Jeff's prototype: https://db2-prototype.netlify.com/

## Testing instructions

* Verify that the README is clear and comprehensive
* Follow the setup directions
* Run `./scripts/test` and make sure you don't get a non-zero exit status
* Run `./scripts/server` and make sure the app is launched and you see the same thing as is shown in the screenshot

Closes https://github.com/PublicMapping/district-builder-2/issues/2
